templates/flamegraph.mustache

```diff
@@ -73,16 +75,22 @@ function init() {
         draw();
     }
 
-    window.excimerChart.label(function(d) {
-        return d.data.name + ' (' + ((d.x1 - d.x0) * 100).toFixed(3)
-                + '%, ' + d.value + ' samples, duration ~' +  d.value * {{samplerate}} + ' ms)'
-    });
+    displaysamplestats = function (d) {
+        time = d.value * {{samplerate}};
+        seconds = time / 1000;
+        minutes = Math.floor(seconds / 60);
+        hours = Math.floor(minutes / 60);
+        minutes = minutes % 60;
+        seconds = (seconds % 60).toFixed(3);
+        durationstr = (hours ? (hours + 'h ') : '') + (minutes ? (minutes + 'm ') : '') + seconds + 's';
 
```

Comment:
It's recommended that we declare variables with `let` or `const` instead of leaving them in the global scope which is the default (See https://stackoverflow.com/a/60610552/4757871).

`let` would be for variables where you expect them to be reassigned / changed.
`const` would be values for which you would not change after setting them initially (for objects this would mean their reference)

Reply:
done